pkg/agent/controller/networkpolicy/cache_test.go

```diff
@@ -170,7 +170,7 @@ func TestRuleCacheAddAddressGroup(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			recorder := newDirtyRuleRecorder()
-			c := newRuleCache(recorder.Record)
+			c := newRuleCache(recorder.Record, []string{"192.168.1.1"})
```

Comment:
maybe you can use a constant for the dummy address in this test as well?

Reply:
I did think about it, but it's in the same package as networkpolicy_controller_test.go, so need to think a new name as I want to avoid using constants from another test file, then I realized the cases in cache_test declares the IP and verifies it in a function scope, and I already declare many other strings there, maybe it's different from the other file declaring the string in one place and not directly in the test function.

Reply:
it was a pretty minor comment, let's merge this as it is